powerspace-json/src/test/scala/com/powerspace/openrtb/powerspace/PowerspaceTestData.scala

```diff
@@ -155,7 +155,7 @@ object PowerspaceTestData {
   }
 
   val bid: Bid = {
-    val ext = BidExt(ctr = Some(0.5d), cpc = Some(0.9d), margin = Some(0.6d), algorithm = Some("random"))
```

Comment:
I think getting the same value for final revenue and for margin is not correct. It can make the test pass even though the code is not right

Reply:
It is the same thing. so from my perspective it is okay.